src/panels/config/helpers/forms/ha-counter-form.ts

```diff
@@ -44,8 +44,8 @@ class HaCounterForm extends LitElement {
     if (item) {
       this._name = item.name || "";
       this._icon = item.icon || "";
-      this._maximum = item.maximum;
-      this._minimum = item.minimum;
+      this._maximum = item.maximum ?? undefined;
+      this._minimum = item.minimum ?? undefined;
```

Comment:
Why is this needed? If it is undefined, we set it to undefined?

Reply:
It comes in as "NaN" but that results in a "0" in the input field. With "undefined" the input stays empty ads intended.